build-linux.sh

```diff
@@ -37,6 +37,11 @@ function main() {
       -t|--test)
         opt_test="true"
         ;;
+       -i|--install-all)
```

Comment:
The intent of my ask for the readme update was to remove the documentation of `-i` from the README, not add it back to the script. I'd rather the scripts have relative parity between platforms.

Please back this change out, and remove the mention of `-i` from the README on the linux section.